src/modules/phasor/generic.cc

```diff
@@ -33,21 +33,22 @@ Generic<OT>::Generic(const Config& config, const Input& input)
     const F64& min_value = max_value * -1.0;
 
     F64 max_cal = 0.0, min_cal = 0.0;
-    for (const auto& calibration : config.antennaCalibrations) {
-        if (calibration.real() > max_cal) {
-            max_cal = calibration.real();
+    for (size_t i = 0; i < config.antennaCalibrations.size(); i++)
```

Comment:
Just curious, the `for (const auto& calibration...` did not work here? I can't think why it shouldn't work. Maybe it's a but in the Vector implementation that can be fixed.

Reply:
Will double check.